--- origsrc/latexila-2.10.1/configure.ac	2014-02-16 07:32:05.000000000 -0600
+++ src/latexila-2.10.1/configure.ac	2014-03-24 23:34:09.647873300 -0500
@@ -120,8 +120,7 @@ LIBS="$LIBS `pkg-config --libs ${depende
 
 # Native Language Support
 IT_PROG_INTLTOOL([0.40.0])
-AM_GNU_GETTEXT_VERSION([0.17])
-AM_GNU_GETTEXT([external])
+AM_GLIB_GNU_GETTEXT
 
 GETTEXT_PACKAGE=${PACKAGE_TARNAME}
 AC_SUBST([GETTEXT_PACKAGE])
